addons/mail/controllers/gif.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.http import request, route, Controller
+import requests
+import json
+
+
+class MailGifController(Controller):
+
+    ckey = "my_test_app"
+    limit = 8
+    # This is the equivalent of PG-13
+    # see https://developers.google.com/tenor/guides/content-filtering
+    contentFilter = "low"
+    tenor_url = "https://tenor.googleapis.com"
```

Comment:
I believe those should become system parameters?

Reply:
Tu client_key ckey can just be the database name, it's just a key for tenor to use.
I converted the others to settings